app/models.py

```diff
@@ -46,9 +126,7 @@ def add_rsvp(self, user):
 
     def get__all_rsvp(self, user):
```

Comment:
the docstring does not reflect what the method does. the method, as it is, will not return `all` but one `user`.